src/pages/about/index.tsx

```diff
@@ -26,7 +26,7 @@ function InfoBlock(props: InfoBlockProps): JSX.Element {
 
 export default function PageAbout(): JSX.Element {
     // query site metadata for page content
-    const query = useStaticQuery(graphql`
+    const query = useStaticQuery<GatsbyTypes.AboutQueryQuery>(graphql`
```

Comment:
This can be removed now as the plugin to do this was uninstalled. They should no be auto added anymore